presto-kafka/src/test/java/io/prestosql/plugin/kafka/TestKafkaDistributed.java

```diff
@@ -14,38 +14,37 @@
 package io.prestosql.plugin.kafka;
 
 import io.airlift.tpch.TpchTable;
-import io.prestosql.plugin.kafka.util.EmbeddedKafka;
+import io.prestosql.plugin.kafka.util.TestingKafka;
 import io.prestosql.testing.AbstractTestQueries;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.Test;
 
 import java.io.IOException;
 
 import static io.prestosql.plugin.kafka.KafkaQueryRunner.createKafkaQueryRunner;
-import static io.prestosql.plugin.kafka.util.EmbeddedKafka.createEmbeddedKafka;
 
 @Test
 public class TestKafkaDistributed
         extends AbstractTestQueries
 {
-    private final EmbeddedKafka embeddedKafka;
+    private final TestingKafka testingKafka;
 
     public TestKafkaDistributed()
             throws Exception
     {
-        this(createEmbeddedKafka());
+        this(new TestingKafka());
     }
 
-    public TestKafkaDistributed(EmbeddedKafka embeddedKafka)
+    public TestKafkaDistributed(TestingKafka testingKafka)
```

Comment:
Use this opportunity to convert this into `getQueryRunner()` method

Reply:
I guess you meant `createQueryRunner`